entry_tasks/89_sequence_type_check/main.py

```diff
@@ -0,0 +1,49 @@
+def parse_value(string_value):
+    if string_value.isdigit():
+        return int(string_value)
+    elif string_value.strip().replace('.', '').isdigit():
+        return float(string_value)
+    elif string_value.strip().lower() == "false":
+        return False
+    elif string_value.strip().upper() == "False":
+        return False
+    elif string_value.strip().lower() == "true":
+        return True
+    elif string_value.strip().upper() == "True":
+        return True
+    elif string_value.strip().lower() == "none":
+        return None
+    elif string_value.strip().upper() == "None":
+        return None
+    elif string_value.__str__():
+        if "(" in string_value and ")" in string_value:
+            array_tuple = string_value.replace("(", "").replace(")", "").replace(" ", "").split(",")
+            for idx, item in enumerate(array_tuple):
+                array_tuple[idx] = parse_value(item)
+            return tuple(array_tuple)
+        else:
+            return str(string_value.strip())
```

Comment:
since string_value is always str and you first do .strip() which also return a str, do not need to convert it once again to str()